### PR TITLE
Fix es and es-MX sign_in and sign_up translation:

### PR DESCRIPTION
**Pull request by roramirez**
This change fix the use the sign_up and sign_in meaning. In the last
version of these files it's was using the reverse meaning.

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -143,8 +143,8 @@ es:
         title: "Reenviar instrucciones de confirmación"
         submit: "Reenviar instrucciones de confirmación"
       links:
-        sign_up: "Ingresar"
-        sign_in: "Registrarse"
+        sign_up: "Registrarse"
+        sign_in: "Ingresar"
```
Comment (javierjulio) [inline config/locales/es.yml:147]: @roramirez I realize this was using `"Ingresar"` before but overall it seems we use `"Iniciar Sesión"` consistently. What do you think about changing that here to the following?
```suggestion
        sign_in: "Iniciar Sesión"
```

Comment (roramirez) [reply config/locales/es.yml:147]: I'm seems good to change this string. It's more clearly to use this way  in spanish

Comment (javierjulio) [reply config/locales/es.yml]: Great thanks!

**Comment by deivid-rodriguez** [comment]
CI failure seems unrelated, so this can be merged I believe?

**Comment by javierjulio** [comment]
@deivid-rodriguez thanks for the heads up! I didn't know it ended up failing. Yes I think so. I checked CI and re-ran the build to ensure its all good and it passed. 👍🏻

**Comment by deivid-rodriguez** [comment]
Thanks!!

**Pull request merged**